services/ingestion/etl/agri/csv/src/lib/uploadFromStream.js

```diff
@@ -23,17 +24,15 @@ export default ({ key, BUCKET, s3, sns, endpointArn, onError, callback }) => {
      * Send the SNS message
      */
     return sns.publish(
-      {
-        Message: JSON.stringify({
-          default: JSON.stringify({
-            key,
-            status: STATUS.PARSED,
-            message: 'ETL successful',
-          }),
-        }),
-        MessageStructure: 'json',
-        TargetArn: endpointArn,
-      },
+      prepareMessage(
+        {
+          key,
+          status: STATUS.PARSED,
+          message:
+            'ETL successfully uploaded a parsed (harmonized) version of incoming data!',
```

Comment:
`ETL successfully uploaded a parsed...` is confusing. Furthermore, I think it could be great if we only publish a code and then translate this code in the frontend with the user's language.

```
status: STATUS.PARSED,
code: 123,
```

and then in the front: `code === 123 -> 'Message in the user's current language'`

Reply:
Good idea!